rubix-client/pom.xml

```diff
@@ -65,11 +65,10 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
-                <version>2.3.2</version>
+                <version>3.1.0</version>
```

Comment:
Do we need this version change?

Reply:
2.3.2 is also working fine. So, will undo the change of version